2022-09-05 11:05:43,638 INFO    MainThread:17455 [wandb_setup.py:_flush():71] setting env: {}
2022-09-05 11:05:43,638 WARNING MainThread:17455 [wandb_setup.py:_flush():71] could not find program at run.py
2022-09-05 11:05:43,638 INFO    MainThread:17455 [wandb_setup.py:_flush():71] setting login settings: {}
2022-09-05 11:05:43,638 INFO    MainThread:17455 [wandb_init.py:_log_setup():371] Logging user logs to /home/zzx/deepke/DeepKE/example/re/standard/logs/2022-09-05_11-05-42/wandb/run-20220905_110543-2zk3yfn6/logs/debug.log
2022-09-05 11:05:43,638 INFO    MainThread:17455 [wandb_init.py:_log_setup():372] Logging internal logs to /home/zzx/deepke/DeepKE/example/re/standard/logs/2022-09-05_11-05-42/wandb/run-20220905_110543-2zk3yfn6/logs/debug-internal.log
2022-09-05 11:05:43,639 INFO    MainThread:17455 [wandb_init.py:init():404] calling init triggers
2022-09-05 11:05:43,639 INFO    MainThread:17455 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2022-09-05 11:05:43,639 INFO    MainThread:17455 [wandb_init.py:init():449] starting backend
2022-09-05 11:05:43,639 INFO    MainThread:17455 [backend.py:_multiprocessing_setup():95] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-09-05 11:05:43,650 INFO    MainThread:17455 [backend.py:ensure_launched():199] starting backend process...
2022-09-05 11:05:43,658 INFO    MainThread:17455 [backend.py:ensure_launched():204] started backend process with pid: 17493
2022-09-05 11:05:43,659 INFO    MainThread:17455 [wandb_init.py:init():458] backend started and connected
2022-09-05 11:05:43,660 INFO    MainThread:17455 [wandb_init.py:init():519] updated telemetry
2022-09-05 11:05:43,674 INFO    MainThread:17455 [wandb_init.py:init():549] communicating current version
2022-09-05 11:05:45,075 INFO    MainThread:17455 [wandb_init.py:init():554] got version response upgrade_message: "wandb version 0.13.2 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-09-05 11:05:45,075 INFO    MainThread:17455 [wandb_init.py:init():564] communicating run to backend with 30 second timeout
2022-09-05 11:05:45,523 INFO    MainThread:17455 [wandb_init.py:init():592] starting run threads in backend
2022-09-05 11:05:47,143 INFO    MainThread:17455 [wandb_run.py:_console_start():1816] atexit reg
2022-09-05 11:05:47,143 INFO    MainThread:17455 [wandb_run.py:_redirect():1690] redirect: SettingsConsole.REDIRECT
2022-09-05 11:05:47,143 INFO    MainThread:17455 [wandb_run.py:_redirect():1695] Redirecting console.
2022-09-05 11:05:47,145 INFO    MainThread:17455 [wandb_run.py:_redirect():1751] Redirects installed.
2022-09-05 11:05:47,146 INFO    MainThread:17455 [wandb_init.py:init():619] run started, returning control to user process
2022-09-05 11:05:47,312 INFO    MainThread:17455 [wandb_run.py:_atexit_cleanup():1786] got exitcode: 1
2022-09-05 11:05:47,313 INFO    MainThread:17455 [wandb_run.py:_restore():1758] restore
2022-09-05 11:05:50,156 INFO    MainThread:17455 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 669
  total_bytes: 669
}

2022-09-05 11:05:50,469 INFO    MainThread:17455 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 669
  total_bytes: 669
}

2022-09-05 11:05:50,570 INFO    MainThread:17455 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 669
  total_bytes: 7434
}

2022-09-05 11:05:50,672 INFO    MainThread:17455 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 669
  total_bytes: 7434
}

2022-09-05 11:05:50,773 INFO    MainThread:17455 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 669
  total_bytes: 7434
}

2022-09-05 11:05:50,874 INFO    MainThread:17455 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 2964
  total_bytes: 7434
}

2022-09-05 11:05:50,976 INFO    MainThread:17455 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 3979
  total_bytes: 7434
}

2022-09-05 11:05:51,077 INFO    MainThread:17455 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7434
  total_bytes: 7434
}

2022-09-05 11:05:51,179 INFO    MainThread:17455 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7434
  total_bytes: 7434
}

2022-09-05 11:05:51,280 INFO    MainThread:17455 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7434
  total_bytes: 7434
}

2022-09-05 11:05:51,381 INFO    MainThread:17455 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7434
  total_bytes: 7434
}

2022-09-05 11:05:51,483 INFO    MainThread:17455 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7434
  total_bytes: 7434
}

2022-09-05 11:05:52,022 INFO    MainThread:17455 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7434
  total_bytes: 7434
}

2022-09-05 11:05:52,652 INFO    MainThread:17455 [wandb_run.py:_wait_for_finish():1916] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7434
  total_bytes: 7434
}
local_info {
}

2022-09-05 11:05:53,764 INFO    MainThread:17455 [wandb_run.py:_append_files():2184] logging synced files
